openassessment/xblock/static/js/src/adg/lib/table_builder.js

```diff
@@ -290,6 +290,19 @@ $.fn.oraTableBuilder = function ($config) {
             $(table).find('td').each(function () {
                 $(this).empty();
                 $(this).append($(_this.controls[value]).clone(true, true));
+
+                if($.browser.safari) {
+                    const checkbox = $(this).find('input');
```

Comment:
Let's please be as specific as possible in the selector, if we want to select only check boxes then change the selector accordingly.